src/lib/components/composites/input/ChipsInput.svelte

```diff
@@ -0,0 +1,214 @@
+<script lang="ts">
+    import { Label } from "$lib/components/primitives/label";
+    import { Button } from "$lib/components/primitives/button";
+
+    interface ChipsInputProps {
+        items: string[];
+        label?: string;
+        labelPosition?: "top" | "left";
+        searchSuggestions?: (arg0: string) => string[];
+        placeholder?: string;
+    }
+
+    let {
+        items = $bindable(),
+        label,
+        labelPosition = "top",
+        searchSuggestions,
+        placeholder = "",
+    }: ChipsInputProps = $props();
+
+    /*
+        Index (from 0 - <length of items>) indicating, which chip is currently selected.
+        Index -1 represent the state, that no chip is selected.
+    */
+    let selectedChipIndex: number = $state(-1);
+    let inputText: string = $state("");
+
+    let suggestions: string[] = $state([]);
+    /*
+        Index (from 0 - <length of suggestions>) indicating, which suggestion is currently selected.
+        Index -1 represent the state, that no suggestion is selected.
+    */
+    let selectedSuggestionIndex: number = $state(-1);
+
+    /**
+     * Adds the input to the list of items and add a chip to the visualization,
+     * if the input is not a duplicate.
+     *
+     * Furthermore, it clears the input.
+     *
+     * @param item the item to add
+     */
+    function addItem(item: string): void {
+        if (item !== "" && !items.includes(item)) {
```

Comment:
Let's add `trim()` so it can't be blank either.
```suggestion
        if (item.trim() !== "" && !items.includes(item)) {
```

Reply:
Not necessary, as I trim the input, before I pass it to this method, but I will change it, as your solution seems better to me.